styles.css

```diff
@@ -10,6 +10,10 @@ body{
 header, footer, aside, .main{
   padding: 1em;
 }
+h1, h3, p, h2, {
```

Comment:
very close, but this misses the `<div>` elements that are a children of `<main>`

a better way would use the child selector like this `header > *, footer > *, .main > *, aside > *`